tests/countries/test_equatorial_guinea.py

```diff
@@ -0,0 +1,305 @@
+        self.assertHolidayName(f"{name} (observado)", obs_dt)
+        self.assertNoNonObservedHoliday(obs_dt)
+
+    def test_independence_day(self):
+        name = "Día de la Independencia Nacional"
+        self.assertHolidayName(name, (f"{year}-10-12" for year in range(2007, 2050)))
+
+        obs_dt = (
+            "2008-10-13",
+            "2013-10-14",
+            "2014-10-13",
+            "2019-10-14",
+            "2024-10-14",
+            "2025-10-13",
+        )
+        self.assertHolidayName(f"{name} (observado)", obs_dt)
+        self.assertNoNonObservedHoliday(obs_dt)
+
+    def test_immaculate_conception(self):
+        name = "Festividad de la Inmaculada Concepción de María"
+        self.assertHolidayName(name, (f"{year}-12-08" for year in range(2007, 2050)))
+
+        obs_dt = (
+            "2007-12-10",
+            "2012-12-10",
+            "2013-12-09",
+            "2018-12-10",
+            "2019-12-09",
+            "2024-12-09",
+        )
+        self.assertHolidayName(f"{name} (observado)", obs_dt)
+        self.assertNoNonObservedHoliday(obs_dt)
+
+    def test_christmas_day(self):
+        name = "Día de Navidad"
+        self.assertHolidayName(name, (f"{year}-12-25" for year in range(2007, 2050)))
+
+        obs_dt = (
+            "2010-12-27",
+            "2011-12-26",
+            "2016-12-26",
+            "2021-12-27",
+            "2022-12-26",
+        )
+        self.assertHolidayName(f"{name} (observado)", obs_dt)
+        self.assertNoNonObservedHoliday(obs_dt)
+
+    def test_subdivision_holidays(self):
+        subdiv_holidays_mapping = {
+            "AN": ("2025-06-13",),
+            "Akurenam": ("2025-12-04",),
+            "Akonibe": ("2025-08-21",),
+            "Anisok": ("2025-06-29",),
+            "Ayene": ("2025-08-15",),
+            "Baney": ("2025-07-25",),
+            "Bata": ("2025-07-25",),
+            "Bidja-Bidjan": ("2025-09-14",),
+            "Bikurga": ("2025-08-15",),
+            "Bitika": ("2025-10-15",),
+            "Ebebiyin": ("2025-09-09",),
+            "Kogo": ("2025-07-15",),
+            "Luba": ("2025-04-27",),
+            "Machinda": ("2025-12-30",),
+            "Malabo": ("2025-11-17",),
+            "Mbini": ("2025-08-22",),
+            "Mikomeseng": ("2025-08-05",),
+            "Mongomeyen": ("2025-12-30",),
+            "Mongomo": ("2025-12-12",),
+            "Niefang": ("2025-08-22",),
+            "Nkimi": ("2025-07-25",),
+            "Nkue": ("2025-12-03",),
+            "Nsok-Nsomo": ("2025-09-09",),
+            "Nsork": ("2025-06-29",),
+            "Rebola": ("2025-04-27",),
+        }
+
+        for subdiv, holidays in subdiv_holidays_mapping.items():
+            self.assertHoliday(EquatorialGuinea(subdiv=subdiv), holidays)
+
+    def test_afcon_victory(self):
+        name = "Victoria de la AFCON contra Costa de Marfil"
+        self.assertHolidayName(name, "2024-01-23")
+        self.assertNoHoliday("2023-01-23", "2025-01-23")
```

Comment:
```suggestion
    def test_special_holidays(self):
        self.assertHoliday(
            "2024-01-23",
        )
```
And this method should be placed after `test_no_holidays()`.